frontends/java/README.md

```diff
@@ -17,7 +43,11 @@ It requires the target source code compiled and packed into jar file.
 
 The resulting call tree are shown in stdout.
 
-Example of running: `java -jar javacg-0.1-SNAPSHOT-static.jar <TARGET_JAR_FILE>`
+Command: `java -jar javacg-0.1-SNAPSHOT-static.jar <TARGET_JAR_FILE>`
+
+Example for execution using testcase test1: `java -jar javacg-0.1-SNAPSHOT-static.jar path/to/fuzz-introspector/tests/java/test1/test1.jar`
```

Comment:
I think we need complete instructions here including getting `javacg-0.1-SNAPSHOT-static.jar`

Reply:
Added with details description.